client/question/components/CodeSnippetDirective.js

```diff
@@ -23,11 +23,11 @@ tie.directive('codeSnippet', [function() {
       getContent: '&content'
     },
     template: `
-      <span ng-repeat="line in abbreviatedSnippetLines track by $index">
+      <span ng-repeat="line in snippetLines track by $index">
         <span class="tie-code-snippet-line">{{line}}</span>
         <br>
       </span>
-      <span ng-if="abbreviatedSnippetLines.length > MAX_NUM_LINES_IN_ABBREVIATED_SNIPPET">
+      <span ng-if="codeLines.length > MAX_NUM_LINES_IN_SNIPPET">
```

Comment:
Just to confirm, abbreviatedSnippetLines is replaced by codeLines here but it's replaced by snippetLines elsewhere?

Reply:
Yes, that was intended; I think the old code was less clear. Does this version make sense to you (logically)?

Reply:
I see. In that case, it seems the older code was actually different? I think the newer version makes sense.

Reply:
Yes, that's right. Thanks!